DevoxxClientMobile/src/main/java/com/devoxx/views/ConfSelectorPresenter.java

```diff
@@ -26,214 +26,115 @@
-    @FXML
-    private Button countryButton;
-    @FXML
-    private Label dateLabel;
-    @FXML
-    private Label venueLabel;
-    @FXML
-    private Label daysLabel;
-    @FXML
-    private Label talksLabel;
-    @FXML
-    private Label capacityLabel;
+    private StackPane filterPane;
 
     @Inject
     private Service service;
+    private ObservableList<Conference> conferences = FXCollections.observableArrayList();
 
     public void initialize() {
-        GluonObservableList<Conference> conferences = service.retrieveConferences();
-
-        countryButton.setMaxWidth(Double.MAX_VALUE);
-        Bindings.bindContent(selector.getItems(), conferences);
-
-        selector.selectedItemProperty().addListener(e -> {
-            Conference c = selector.getSelectedItem();
-
-            // TODO: add the splash image somehow as the background image over the entire view??
-            // ImageView background = new ImageView(new Image(c.getSplashImgURL()));
-
-            countryLabel.setText(c.getCountry());
-
-            dateLabel.setText(MessageFormat.format(bundle.getString("OTN.CONFERENCE_SELECTOR.INFO.DATE"),
-                    c.getStartDate().format(DATE_FORMATTER_SHORT), c.getEndDate().format(DATE_FORMATTER_LONG)));
-
-            long days = c.getDaysUntilStart();
-            daysLabel.setText(days > 0 ? String.valueOf(days) : "0");
-            talksLabel.setText(c.getSessions());
-            capacityLabel.setText(c.getCapacity());
-            venueLabel.setText(c.getVenue() + ", " + c.getCountry());
-        });
-
-        Optional<SettingsService> settingsService = Services.get(SettingsService.class);
-        // If saved conference is found select it else select the nearest conference
+        final Optional<SettingsService> settingsService = Services.get(SettingsService.class);
         if (settingsService.isPresent()) {
-            SettingsService service = settingsService.get();
-            String savedConference = service.retrieve(DevoxxSettings.SAVED_CONFERENCE_ID);
-            if (savedConference == null || savedConference.isEmpty()) {
-                selectNearestConferenceFrom(conferences);
+            final SettingsService settings = settingsService.get();
+            final String eventType = settings.retrieve(DevoxxSettings.SAVED_CONFERENCE_TYPE);
+            if (eventType == null || eventType.equals("")) {
+                Bindings.bindContent(conferences, service.retrieveConferences(Conference.Type.DEVOXX));
             } else {
-                selectSavedConferenceFrom(conferences, savedConference);
+                Bindings.bindContent(conferences, service.retrieveConferences(Conference.Type.valueOf(eventType)));
             }
         } else {
-            selectNearestConferenceFrom(conferences);
+            Bindings.bindContent(conferences, service.retrieveConferences(Conference.Type.DEVOXX));
         }
 
+        Bindings.bindContent(selector.itemsProperty(), conferences);
```

Comment:
Is there a reason for using the intermediate `conferences` list? There is now a double list content binding from `selector.items` to `conferences` and then from `conferences` to `service.retrieveConferences`. When switching between the conference types (devoxx and voxxed), I sometimes get the following exception:

    08-28 16:08:25.920 19581 19610 E AndroidRuntime: java.lang.IndexOutOfBoundsException
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at java.util.AbstractList.subList(AbstractList.java:738)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at javafx.collections.ModifiableObservableListBase.subList(ModifiableObservableListBase.java:189)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.binding.ContentBinding$ListContentBinding.onChanged(ContentBinding.java:114)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.collections.ListListenerHelper$SingleChange.fireValueChangedEvent(ListListenerHelper.java:163)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.collections.ListListenerHelper.fireValueChangedEvent(ListListenerHelper.java:72)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at javafx.collections.ObservableListBase.fireChange(ObservableListBase.java:232)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at javafx.collections.ListChangeBuilder.commit(ListChangeBuilder.java:482)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at javafx.collections.ListChangeBuilder.endChange(ListChangeBuilder.java:541)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at javafx.collections.ObservableListBase.endChange(ObservableListBase.java:204)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at javafx.collections.ModifiableObservableListBase.setAll(ModifiableObservableListBase.java:90)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.gluonhq.impl.cloudlink.client.data.function.GluonCloudRemoteFunctionListDataReaderImpl.lambda$read$1(GluonCloudRemoteFunctionListDataReaderImpl.java:120)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.gluonhq.impl.cloudlink.client.data.function.GluonCloudRemoteFunctionListDataReaderImpl$$Lambda$2.run(GluonCloudRemoteFunctionListDataReaderImpl.java)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.application.PlatformImpl.lambda$null$156(PlatformImpl.java:295)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.application.PlatformImpl.access$lambda$18(PlatformImpl.java)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.application.PlatformImpl$$Lambda$19.run(Unknown Source)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at java.security.AccessController.doPrivileged(AccessController.java:52)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.application.PlatformImpl.lambda$runLater$157(PlatformImpl.java:294)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.application.PlatformImpl.access$lambda$5(PlatformImpl.java)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.javafx.application.PlatformImpl$$Lambda$6.run(Unknown Source)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:93)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:52)
    08-28 16:08:25.920 19581 19610 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:831)

When I changed the code to directly setting the retrieved conferences as items on the selector, I didn't notice any difference in functionality and I no longer get the exception. So instead of:

    Bindings.bindContent(conferences, service.retrieveConferences(Conference.Type.DEVOXX));

I did the following:

    selector.setItems(service.retrieveConferences(Conference.Type.DEVOXX));

The `conferences` variable can then be removed as well.